modules/util/detect.js

```diff
@@ -77,8 +77,8 @@ export function utilDetect(refresh) {
 
   _detected.isMobileWebKit = (/\b(iPad|iPhone|iPod)\b/.test(ua) ||
```

Comment:
Hah I didn't know this was in here.. It looks like we're not actually using this code - the `isMobileWebkit` flag is commented out in `UISystem.js`, so maybe best to remove the whole thing?

If I remember correctly, it was because Safari supports special `Gesture` events that no other browser supports?  UiSystem attempt(ed?) to cancel those events, but it could do that same thing without the `isMobileWebkit` flag. 🤔

Reply:
Thanks for catching that! You're right, the _detected.isMobileWebKit flag isn't being used in UISystem.js so I’ll just remove it.